service_mesh/v2x/ossm-create-smcp.adoc

```diff
@@ -5,6 +5,11 @@ include::modules/ossm-document-attributes.adoc[]
 
 You can deploy a basic installation of the `ServiceMeshControlPlane` by using either the {product-title} web console or from the command line using the `oc` client tool.
 
+[NOTE]
+====
+The {ProductShortName} documentation uses `istio-system` as the example project, but you may deploy the service mesh to any project.
```

Comment:
```suggestion
The {ProductShortName} documentation uses `istio-system` as the example project, but you can deploy the service mesh to any project.
```

> !  may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.

Reply:
Replaced in two files.